### PR TITLE
Improve project application

### PR DESCRIPTION
**Pull request by mattbk**
Simplifying this to just changing the button on the front page.

Including ~~https://github.com/gratipay/gratipay.com/issues/4270,~~ https://github.com/gratipay/gratipay.com/pull/4268#issuecomment-269844784, ~~https://github.com/gratipay/gratipay.com/issues/3859, and onboarding in general.~~

### CODE REVIEW

**Comment by whit537** [comment]
Kicked the other half of travis ...

File: www/index.html.spt [hunk 1]
```diff
@@ -82,7 +82,7 @@ page_id = "homepage"
 {% endif %}
 
 <form action="new" class="apply">
-    <button type="submit">{{ _("Apply to Join") }}</button>
+    <button type="submit">{{ _("Fund your Project") }}</button>
```
Comment (nobodxbodon) [inline www/index.html.spt]: nit: your -> Your

Comment (mattbk) [inline www/index.html.spt]: Changed in e4605e61c7ec907b6b3ec3123602dcb20bd1a274.

**Comment by mattbk** [comment]
Go for it.

**Comment by mattbk** [comment]
!m @nobodxbodon.

**Pull request merged**